src/component.js

```diff
@@ -13,9 +13,19 @@ export default class Component {
     constructor(el, seedDataForCloning = null) {
         this.$el = el
 
-        const dataAttr = this.$el.getAttribute('x-data')
+        let dataAttr = this.$el.getAttribute('x-data')
+
+        if (! dataAttr && this.$el.getAttribute('data-x-data')) {
+            dataAttr = this.$el.getAttribute('data-x-data')
+        }
+
         const dataExpression = dataAttr === '' ? '{}' : dataAttr
-        const initExpression = this.$el.getAttribute('x-init')
+
+        let initExpression = this.$el.getAttribute('x-init')
+
+        if (! initExpression && this.$el.getAttribute('data-x-init')) {
+            initExpression = this.$el.getAttribute('data-x-init')
+        }
```

Comment:
Same
`const initExpression = this.$el.getAttribute('x-init') || this.$el.getAttribute('data-x-init')`